streaming/src/test/scala/org/apache/spark/streaming/UISeleniumSuite.scala

```diff
@@ -163,7 +164,7 @@ class UISeleniumSuite
 
         // Check job ids
         val jobIdCells = findAll(cssSelector( """#batch-job-table a""")).toSeq
-        jobIdCells.map(_.text) should be (List("0", "1", "2", "3"))
+        jobIdCells.map(_.text) should be (List("?", "?", "0", "1", "2", "3"))
```

Comment:
Maybe we can filter out the "?" to make the test a little more robust. It isn't trying to test the tooltips.

Reply:
Tooltips are defined with hyperlink tag in UIUtils.scala. And in this case we are also searching for all the hyperlinks in the table 'batch-job-table'. That's why "?" is required here.

Reply:
I understand, but it's not what this code is trying to test. It'd be more robust if it only tested what it is looking for.

Reply:
Modified the test case to only check for "jobId". Please review and retest.